src/NHSD.BuyingCatalogue.Organisations.Api/Settings/OdsSettings.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace NHSD.BuyingCatalogue.Organisations.Api.Settings
+{
+    internal sealed class OdsSettings
+    {
+        public string ApiBaseUrl { get; set; }
+        public string[] BuyerOrganisationRoleIds { get; set; }
```

Comment:
I think I'd probably use `HashSet<string>` here. Although it's not going to make any noticeable difference in our case where we currently only have two IDs, `Contains` has to iterate through the array whereas the hash set can perform a hash lookup.

However, as it's unlikely to make any difference in our case feel free to leave as it is – it's up to you.

Reply:
I'm all up for small optimisations that can help, however in this instance, as the search space is and will be very narrow - [maximum 5 elements](https://digital.nhs.uk/services/organisation-data-service/guidance-for-developers/search-parameters#primaryroleid-and-nonprimaryroleid)  I believe such change would make no difference, according to [this](https://stackoverflow.com/questions/150750/hashset-vs-list-performance), it would actually be slower! (I'm assuming that operations on arrays are even faster than on List<T>s.)